ebusd/CHANGELOG.md

```
@@ -1,17 +1,71 @@
+## Features
+
+* add "-m" option to "encode" and "decode" commands
+* add output for commands executed with "--inject=stop"
+* add secondary replacement value for date types
+* add value range and step support for numeric types
+* add numeric base types without replacement value (e.g. "U1L", "S1L", "U2L", "S2L", "U2B", "S2B", etc.)
+* add also include write messages as read-only ones in MQTT definition topic if writes are excluded
+
+
+
+# Next Release (tbd)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)